packages/ses/src/ses.d.ts

```diff
@@ -0,0 +1,38 @@
+type safetyPreference = 'safe' | 'unsafe';
+
+type LockdownOpts = {
+  fringeSet?: {
+    add?: Function,
+    has?: Function,
+  },
+  dateTaming?: safetyPreference,
+  errorTaming?: safetyPreference,
+  mathTaming?: safetyPreference,
+  regExpTaming?: safetyPreference,
+
+  [extraOption:string]?: any,
+};
+
+declare var lockdown = (LockdownOpts) => void;
+
+declare var harden = (any) => any;
+
+declare class Compartment {
+  constructor(endowments: Object);
+  evaluate(command: string): any;
+  globalThis: any;
+  import: () => Promise<any>;
```

Comment:
import takes a FullSpecifier string and returns a `Promise<{ namespace: Object }>` specifically.